torch_geometric/distributed/local_graph_store.py

```diff
@@ -18,10 +18,38 @@ def __init__(self):
         self._edge_attr: Dict[Tuple, EdgeAttr] = {}
         self._edge_id: Dict[Tuple, Tensor] = {}
 
+        self.num_partitions: int = 1
+        self.partition_idx: int = 0
+        self.node_pb: Union[torch.Tensor, Dict[NodeType, torch.Tensor]] = None
+        r""" node_partition_book:  mapping between node ids and partition idx """
```

Comment:
Move the comment up. Also use `#` instead of `"""`.

Reply:
done